Assets/Scripts/ProjectileHazardScript.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ProjectileHazardScript : MonoBehaviour
+{
+    [SerializeField] float strengthOfAttraction = 5f;
+    [SerializeField] float pullRadius = 10f;
+    [SerializeField] GameObject playerRocket;
+
+    Vector3 initialRocketPosition;
```

Comment:
Let's move this variable to with the Update() method. This doesn't need to be global.

Also let's change the name because it's misleading. this should be called something like `targetVector`